cli/config.go

```diff
@@ -74,6 +74,7 @@ type hypervisor struct {
 	KernelParams          string `toml:"kernel_params"`
 	MachineType           string `toml:"machine_type"`
 	DefaultVCPUs          int32  `toml:"default_vcpus"`
+	DefaultMaxVCPUs       int32  `toml:"default_maxvcpus"`
```

Comment:
Why signed int?  Is there a scenario where you'd have a negative number number of CPUs?

Reply:
this is part of configuration file, if this value is <= 0 then the actual number of vCPUs will be used

Reply:
my question is why negative needs to be supported?  Seems == 0 should be enough here?

Reply:
yep, it should be enough, do you prefer to have an `uint32` here?

Reply:
I used an `int32` to support negative values and be similar to `default_vcpus`